roboprop/settings.py

```diff
@@ -132,7 +132,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.2/howto/static-files/
 
-STATIC_URL = "static/"
+STATIC_URL = "/static/"
```

Comment:
actually this silly little thing is most likely cause of image issues in k8s. Local systems being more friendly